main.tf

```diff
@@ -118,12 +113,15 @@ resource "kubernetes_deployment" "wandb" {
             value = var.oidc_client_id
           }
 
-          env {
-            name = "OIDC_SECRET"
-            value_from {
-              secret_key_ref {
-                name = local.app_name
-                key  = "OIDC_SECRET"
+          dynamic "env" {
+            for_each = var.other_wandb_secrets
```

Comment:
This is wrong, why did you remove the hard coded static secrets.  If you're gonna do it this way this needs to be `local.all_secrets`...

Reply:
gah, sorry for the confusion, reverted the change from your branch. I kinda prefer using local.all_secrets, so here's stacked PR: https://github.com/wandb/terraform-kubernetes-wandb/pull/36